connector/src/main/resources/config.yml

```diff
@@ -86,6 +86,10 @@ default-locale: en_us
 # Geyser has direct access to the server itself.
 cache-chunks: false
 
+#Allows custom skulls to be displayed. This can cause lag depending on the amount
+#of custom skulls. This requires chunk caching to be enabled to work correctly
+allow-custom-skulls: false
```

Comment:
Is the performance hit significant enough where this is warranted?

Reply:
There should also be a space between `#` and the text to follow the rest of the comments